src/api/track.ts

```diff
@@ -139,6 +150,17 @@ class TrackAPI {
           'X-Radar-Body-Is-Token': 'true',
         },
       });
+
+      if (response && response.user) {
+        if (!response.user.metadata) {
+          response.user.metadata = {};
+        }
+
+        response.user.metadata['radar:debug'] = {
+          sclVal,
+          cslVal,
+        };
```

Comment:
Could wrap this in `options.debug` if its not applicable to all users 

(gets logged when SDK is initialized with `debug: true`)

Reply:
Just added this to print on ping.radar-verify.com (not console). Can remove later. If there's a better way to stash than updating `metadata`, lmk